apps/aechannel/test/aesc_txs_SUITE.erl

```diff
@@ -3412,7 +3412,7 @@ fp_register_name(Cfg) ->
             ?TEST_LOG("Contract create tx ~p", [ContractCreateTx]),
             OnChainTrees = aesc_test_utils:trees(S0),
             TxEnv = tx_env(#{height => 3}),
-            {ok, _} = aetx:check(ContractCreateTx, OnChainTrees,
+            {ok, _} = aetx:process(ContractCreateTx, OnChainTrees,
```

Comment:
It looks like you do the same things twice?

Reply:
Correct. I'll fix it.